src/commands/attest/create.ts

```diff
@@ -0,0 +1,136 @@
+          return ret
+        }
+      }
+    }
+
+    getStatus(): 'passed' | 'failed' {
+      const validPassResponses = new Set(['p', 'passed', 'pass'])
+      const validFailResponses = new Set(['f', 'failed', 'fail', 'failure'])
+      while (true) {
+        const input = prompt('Enter status ((p)assed/(f)ailed): ') || ''
+        if (validPassResponses.has(input.trim().toLowerCase())) {
+          return 'passed'
+        }
+
+        if (validFailResponses.has(input.trim().toLowerCase())) {
+          return 'failed'
+        }
+      }
+    }
+
+    promptForAttestation(id: string): Attestation {
+      return {
+        control_id: id,
+        explanation: this.promptForever('Attestation explanation: '),
+        frequency: this.promptForever('Frequency (1d/3d/1wk/2wk/1m/3m/6m/1y/1.5y/custom): '),
+        status: this.getStatus(),
+        updated: new Date().toISOString(),
+        updated_by: this.promptForever('Updated By: '),
+      }
+    }
+
+    async run() {
+      const {flags} = await this.parse(CreateAttestations)
+
+      const attestations: Attestation[] = []
+
+      if (flags.input) {
+        const evaluation = JSON.parse(fs.readFileSync(flags.input, 'utf8')) as ExecJSON.Execution
+        const search = new AccurateSearch()
+        const controls: Record<string, ExecJSON.Control> = {}
+        for (const profile of evaluation.profiles) {
+          for (const control of profile.controls) {
+            controls[control.id] = control
+            search.addText(control.id, control.id + ': ' + (control.title || '') + ' ' + control.desc || '')
+          }
+        }
+
+        while (true) {
+          const input = prompt("Enter a control ID, search for a control, or enter 'q' to exit: ")
+          if (input.trim().toLowerCase() === 'q') {
+            break
+          } else if (input in controls) {
+            const control = controls[input]
+            const attestation = this.promptForAttestation(control.id)
+            attestations.push(attestation)
+          } else {
+            const ids = search.search(input).slice(0, MAX_SEARCH_RESULTS)
+            ids.forEach((id: string) => {
+              const control = controls[id]
+              console.log(`\t${control.id}: ${control.title?.replace(/\n/g, '').replace(/\s\s+/g, ' ')}`)
+            })
+          }
+        }
+      } else {
+        while (true) {
+          const input = prompt("Enter a control ID or enter 'q' to exit: ")
+          if (input.trim().toLowerCase() === 'q') {
+            break
+          } else {
+            attestations.push(this.promptForAttestation(input))
+          }
+        }
+      }
+
+      switch (flags.format) {
+      case 'json': {
+        fs.writeFileSync(flags.output, JSON.stringify(attestations, null, 2))
+        break
+      }
+
+      case 'xlsx': {
+        XlsxPopulate.fromDataAsync(dataURLtoU8Array(files.AttestationTemplate.data)).then((workBook: any) => {
+          const sheet = workBook.sheet(0)
```

Comment:
probably good to leave a comment that sheet 0 is intended to be that explicitly named attestations sheet